packages/paypal-commerce-integration/src/paypal-commerce-credit-card/paypal-commerce-credit-cards-payment-strategy.ts

```diff
@@ -571,24 +413,59 @@ export default class PayPalCommerceCreditCardsPaymentStrategy implements Payment
         return omitBy(objectWithKebabCaseKeys(styles), isNil);
     }
 
-    private getValidStyleString(styles: HostedInputStyles = {}): string {
-        const validStyles = this.mapStyles(styles);
-
-        return Object.keys(validStyles)
-            .map((key) => `${key}: ${validStyles[key]}`)
-            .join(';');
-    }
-
     /**
      *
      * Utils
      *
      */
-    private getHostedFieldsOrThrow(): PayPalCommerceHostedFields {
-        if (!this.hostedFields) {
+    private getCardFieldsOrThrow(): PayPalCommerceCardFields {
+        if (!this.cardFields) {
             throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
         }
 
-        return this.hostedFields;
+        return this.cardFields;
+    }
+
+    private getInputStyles(
+        styles?: HostedFieldStylesMap,
+    ): PayPalCommerceHostedFieldsRenderOptions['styles'] {
+        const defaultStyles = {
+            input: {
+                'font-size': '1rem',
+                'font-family': 'Montserrat, Arial, Helvetica, sans-serif',
+                color: '#5f5f5f',
+                outline: 'none',
+                padding: '9px 13px',
+            },
+        };
+        const mappedStyles = styles ? this.mapStyleOptions(styles) : null;
+
+        return mappedStyles
+            ? {
+                  ...mappedStyles,
+                  input: {
+                      ...mappedStyles.input,
+                      outline: 'none',
+                      padding: '9px 13px',
+                  },
+              }
+            : defaultStyles;
+    }
+
+    private stylizeInputContainers(fields: HostedCardFieldOptionsMap): void {
+        const { cardNumber, cardExpiry, cardCode, cardName } = fields || {};
+
+        [cardNumber, cardExpiry, cardCode, cardName].forEach((id) => {
```

Comment:
Maybe would be better to use `Object.values(fields || {}).forEach(...)` here to avoid redundant destructurisation in this case. What do you think?